docs/howto/manage-charms.rst

```diff
@@ -234,12 +234,17 @@ charm's project file, specify an item under the :ref:`links.website
 Add docs
 ~~~~~~~~
 
-If you publish your charm on Charmhub, reference documentation about the charm's
-resources, actions, configurations, relations, and libraries is extracted automatically.
-However, you should also aim to add further docs, e.g., a tutorial, how-to guides, etc.
-To provide a link to these docs, in your charm's project file, specify a
-value for the :ref:`links.documentation <charmcraft-yaml-key-documentation>` key. Note
-that at present this must be a Discourse page. E.g.,
+When you publish your charm on Charmhub, reference documentation about the charm's
+resources, actions, configurations, relations, and libraries is generated and 
+published automatically in respective tabs.
+Use the Description tab to add additional documentation, such as tutorials or 
+how-to guides, focusing specifically on the charm itself. 
+For workload-specific or Juju-related content, 
+link to the appropriate upstream documentation.
+
+To add content to the Description tab, create a Discourse topic and include its URL 
+in your charm's project file under the 
+:ref:`links.documentation <charmcraft-yaml-key-documentation>` key:
```

Comment:
Keep in mind I've never managed a charm, but I feel like these two paragraphs step on each other. Both describe adding content on the tab but in different ways. Should they be combined?

Reply:
Agreed - combined witch your previous suggestion of putting this in a new paragraph that flows better IMO.

Reply:
I've experimented with it a bit. Please check the [new version](https://github.com/canonical/charmcraft/pull/2253/commits/8ef59e0da29fd0535a9d45fe03fa8ec90af3ef13)